README.md

```diff
@@ -0,0 +1,23 @@
+
+## Phoenicis PlayOnLinux
+
+Phoenicis PlayOnLinux is a graphical front-end for Wine.
+Install and run Windows software on Linux
+
+
+## Installation Instruction's :
+
+#### Install Extension
+```
+flatpak install flathub org.freedesktop.Platform.Compat.i386
+```
+
+#### Install Application
+```
+flatpak install flathub org.phoenicis.playonlinux
+```
+
+## Run :
```

Comment:
What is the point of spaces before colons?

Reply:
is that so important ?
grammer and typo's i can understand ... but simple formatting ... is that so  much of a concern ?
I am no coder ... but will a space cause any problem in the functionality of the application ?

Reply:
@aiamuzz It looks bad and unprofessional.

Reply:
@AsciiWolf ... seriously ... now we are going to debate over it !!!
bad & unprofessional ... since when did layout / formatting become unprofessional ... 

![deepinscreenshot_select-area_20190124130952](https://user-images.githubusercontent.com/3283676/51662338-83233e00-1fd9-11e9-9004-dfcb99e222d9.png)

If you tell me why there are spaces between the `+` and the text  ? 
And i'll tell you why the space precedes the colon ...

Dude ... its a simple README file ... your comments about why spaces here why spaces there is a little overboard ... wasting your time and mine ...

Reply:
> If you tell me why there are spaces between the + and the text ?

There are no any spaces there. You may look at document preview here: https://github.com/flathub/org.phoenicis.playonlinux/blob/f54bb34d3cca7bc0710d97bb8093d95b5638bc49/README.md

> Dude ... its a simple README file ... your comments about why spaces here why spaces there is a little overboard ... wasting your time and mine ...

With such attitude you are wasting everybody time with this PR as no one will accept work from someone who refuses to take even trivial advice. Please reconsider this.

Reply:
helping fix langauage/grammer is sure welcome and i did acknowledge it ... but don't you think you getting stuck on a 'space' is overboard ? ...

for one you certainly didn't answer my question, on why there are spaces on both sides of the '+' symbol(in the image i have posted above) ... care to answer it ?

Reply:
@aiamuzz You are being extremely unhelpful. English isn't my mother tongue either and I always appreciate if someone points out any mistakes I made. There is no reason to get so emotional about it.

Reply:
I agree this is petty much overboard as removing that space takes a second and nobody expected any dispute about this.

I already answered your question. There are no single _space_ character in position you mentioned. Seeing any gap there which comes from web page formatting doesn't mean there is a _space_ (I hope you don't format your documents with spaces).

Reply:
The space between "+" and the contents is just how github renders unified diff. If you don't like this, bring this up with github. It has nothing to do with flatpak.

Reply:
@Erick555 ... Its not about attitude ... its about accepting what is a worthy feedback ... and what isn't ... and ignoring the isn't ...

@AsciiWolf ... spoke about the space once, i considered it for a little while, then thought, it wasn't significant enough to consider ... so i ignored it ... then he raised the same issue of 'space before a colon' again with a remark !!!

Firstly this is a simple layout & formatting of text ... not really of any importance to the application functionality ... that we are having such a deep debate about it ...

Secondly ever heard of the term 'kerning' ... google it ... you'll know the reason ... i design fonts as a profession, for me spacing between glyphs is important ... so in my professional capacity if i took a call on the spacing feedback and thought it was best to ignore ...

and when somebody continues to harp on the same thing or remark about it ... things do tend to blow up a little !!!

Reply:
> The space between "+" and the contents is just how github renders unified diff. If you don't like this, bring this up with github. It has nothing to do with flatpak.

wow ... @alexlarsson ... hahahaha ... my question about the "+" was not an objection but more of a way to answer to @AsciiWolf 's question about spacing ... 

exactly ... its the way github renders it or the person who design that UI formatted it in that way, as he saw best as the UI designer of that module ... and being a designer myself ... you know why i may have preferred a space before the colon.

I just feel foolish having to defend a 'space' ... on an relatively insignificant issue.

i don't know about us ... but the " **space** " before my colon sure must be feeling on top of the world ... as so many people are debating and discussing it !!! 

:)

Reply:
There aren't many people on the world who feel need to defend putting space before a colon when everyone recommend against it, so it's not surprising that people are linking this issue in various corners of the internet which is making you pretty famous 😄 .

Reply:
@aiamuzz ... wish i did ... but my ... **space before the colon** ... that's actually become famous ... Damn !!! ... thanks to @AsciiWolf ... :cry: